plugins/book-data-extension/src/plugin/index.ts

```diff
@@ -1,2 +1,49 @@
-export const dunnystring =
-	"This is a dummy string so that the 'build' script doesn't fail (warn = fail in CI) due to empty bundle. Remove when there's something to build.";
+import { BookFetcherPlugin as BookFetcherPluginType, BookEntry } from "@librocco/db";
+import { postMessage, addEventListener, removeEventListener } from "./window-helpers";
+import { listenForBooks } from "./listeners";
+
+export const BookFetcherPlugin = (): BookFetcherPluginType => {
+	const fetchBookData = async (isbns: string[]): Promise<BookEntry[]> => {
+		// Check if the extension is registered if not resolve to []
+
+		postMessage(`BOOK_FETCHER_REQ:checkForExtension:${isbns}`);
```

Comment:
1. Can we use a consistent casing/format in messages?
2. Can we use shorter messages? 
3. Why send isbns when checking that the extension is available?

In this case `BOOK_FETCHER:PING` would suffice